19/src/main.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { useState } from "react";
+import _ from "lodash";
```

Comment:
Lodash is not a big library, but in general, it contains many methods that you don't actually need.
A better solution would be to import it like this:
```js
import orderBy as _orderBy from 'lodash/orderBy'
```